src/main/java/com/salesforce/dataloader/mapping/LoadMapper.java

```diff
@@ -88,8 +92,8 @@ public void verifyMappingsAreValid() throws MappingInitializationException {
         for (Map.Entry<String, String> entry : getMappingWithUnmappedColumns(false).entrySet()) {
             String sfdcName = entry.getValue();
             if(StringUtils.hasText(sfdcName)) {
-                final Field f = getClient().getField(sfdcName);
-                if (f == null)
+                final HashSet<Field> fieldSet = getClient().getField(sfdcName);
+                if (fieldSet == null | fieldSet.isEmpty())
```

Comment:
That's a good question.  I don't think I've been able to get the test to 
run.  When I make it locally I have to -DskipTests. Honestly, I'm more 
of a Smalltalker and Apex programmer than Java. I needed help just to 
get the Maven project into Eclipse!

I'll try again to see if they'll run.

On 4/28/15 7:11 PM, neetikumar wrote:
>
> In src/main/java/com/salesforce/dataloader/mapping/LoadMapper.java 
> <https://github.com/forcedotcom/dataloader/pull/77#discussion_r29297985>:
>
> > @@ -88,8 +92,8 @@ public void verifyMappingsAreValid() throws MappingInitializationException {
> >          for (Map.Entry<String, String> entry : getMappingWithUnmappedColumns(false).entrySet()) {
> >              String sfdcName = entry.getValue();
> >              if(StringUtils.hasText(sfdcName)) {
> > -                final Field f = getClient().getField(sfdcName);
> > -                if (f == null)
> > +                final HashSet<Field> fieldSet = getClient().getField(sfdcName);
> > +                if (fieldSet == null | fieldSet.isEmpty())
>
> Would you be able to write some tests for this
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/forcedotcom/dataloader/pull/77/files#r29297985>.
>